test/io.jl

```diff
@@ -384,4 +384,19 @@ module TestIO
     # Enforces matching column count if append == true
     writetable(tf, df3)
     @test_throws DimensionMismatch writetable(tf, df3b, header = false, append = true)
+
+    # Quotemarks are escaped
+    tf = tempname()
```

Comment:
AFAICT you don't need to repeat these lines. Just do as tests above which overwrite the contents of that file.